packages/backend/src/controllers/tasks/dto/create-task.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { TaskModel } from '../../../db/task/task.schema';
+
+export class CreateTaskDto extends TaskModel {
+  @ApiProperty()
+  readonly name: string;
+
+  @ApiProperty()
+  readonly description: string;
+
+  @ApiProperty()
+  readonly due_date: Date;
```

Comment:
are we electing for snake case or cammel? if we don't have any warnings for this we can add it to our linter